packages/orchestration/src/examples/stakeBld.contract.js

```diff
@@ -34,48 +40,86 @@ export const start = async (zcf, privateArgs, baggage) => {
     baggage,
     privateArgs.marshaller,
   );
-  const makeLocalchainAccountKit = prepareLocalchainAccountKit(
+
+  // Mocked until #8879
+  // Would expect this to be instantiated elsewhere, and passed in as a reference
+  const agoricChainInfo = prepareMockChainInfo(zone);
+
+  const timerBrand = await E(privateArgs.timerService).getTimerBrand();
+  const timestampHelper = makeTimestampHelper(
+    privateArgs.timerService,
+    timerBrand,
+  );
+
+  const makeLocalChainAccountKit = prepareLocalChainAccountKit(
     baggage,
     makeRecorderKit,
     zcf,
+    timestampHelper,
+    agoricChainInfo,
   );
 
-  const publicFacet = zone.exo('StakeBld', undefined, {
-    makeStakeBldInvitation() {
-      return zcf.makeInvitation(
-        async seat => {
-          const { give } = seat.getProposal();
-          trace('makeStakeBldInvitation', give);
-          // XXX type appears local but it's remote
-          const account = await E(privateArgs.localchain).makeAccount();
-          const lcaSeatKit = zcf.makeEmptySeatKit();
-          atomicTransfer(zcf, seat, lcaSeatKit.zcfSeat, give);
-          seat.exit();
-          trace('makeStakeBldInvitation tryExit lca userSeat');
-          await E(lcaSeatKit.userSeat).tryExit();
-          trace('awaiting payouts');
-          const payouts = await E(lcaSeatKit.userSeat).getPayouts();
-          const { holder, invitationMakers } = makeLocalchainAccountKit(
-            account,
-            privateArgs.storageNode,
-          );
-          trace('awaiting deposit');
-          await E(account).deposit(await payouts.In);
+  async function makeLocalAccount() {
+    const account = await E(privateArgs.localchain).makeAccount();
+    const address = await E(account).getAddress();
+    return makeLocalChainAccountKit({
+      account,
+      address,
+      storageNode: privateArgs.storageNode,
+    });
+  }
 
-          return {
-            publicSubscribers: holder.getPublicTopics(),
-            invitationMakers,
-            account: holder,
-          };
-        },
-        'wantStake',
-        undefined,
-        M.splitRecord({
-          give: { In: bldAmountShape },
-        }),
-      );
+  const publicFacet = zone.exo(
+    'StakeBld',
+    M.interface('StakeBldI', {
+      makeAccount: M.callWhen().returns(M.remotable('LocalChainAccount')),
+      makeAcountInvitationMaker: M.call().returns(M.promise()),
+      makeStakeBldInvitation: M.call().returns(M.promise()),
```

Comment:
I think this way it'll validate the return value. It at least improves the documentation. We should use this style so when we start inferring TS types from the Pattern the info is there.
```suggestion
      makeAcountInvitationMaker: M.callWhen().returns(InvitationShape),
      makeStakeBldInvitation: M.callWhen().returns(InvitationShape),
```